elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -87,16 +107,20 @@ $LOCAL-VARIABLES: (
 	}
 
 	&__content {
-		overflow-y: auto;
+		overflow-y: pfe-local(Overflow, $region: overlay);
 		overscroll-behavior: contain;
-		max-height: pfe-local(MaxHeight);
+		height: pfe-local(MaxHeight);
```

Comment:
What's the need for changing from `max-height` to `height`? All modals take up 90% of the screen height now?

Reply:
i can't remember.. resetting it back